### PR TITLE
Description markdown editor

### PR DESCRIPTION
**Pull request by jozzi**
CSV 'spec.description' is splitted into 3 separate parts in our data model and edited using 3 editors.
Merged together when yaml is created out of data model

Replaced old PR as rebase with force push was required and PR history was completely polluted

Squashed commit:

[40a0968] CSV 'spec.description' is now separated into 3 sections and edited separately
Validation is separated as well on editor level to highlight which description violates criteria
fixed linter errors

[8e2d535] normalized line ending so it workes correctly on windows

### CODE REVIEW

File: frontend/src/styles/operator-editor-page.scss [hunk 1]
```diff
@@ -100,7 +100,6 @@
 
     &__status {
       .fa, .pficon {
-        margin-left: 10px;
```
Comment (jeff-phillips-18) [inline frontend/src/styles/operator-editor-page.scss]: Looks like this is a rebase issue and shouldn't be changed here.

File: frontend/src/styles/operator-editor-page.scss [hunk 2]
```diff
@@ -511,8 +510,8 @@
       }
 
       &__actions {
-        .oh-button {
-          margin: 0 10px;
+        .oh-operator-editor-page__section__edit-button {
+          margin-right: 10px;
         }
```
Comment (jeff-phillips-18) [inline frontend/src/styles/operator-editor-page.scss]: Looks like this is a rebase issue and shouldn't be changed here.

File: frontend/src/styles/operator-editor-page.scss [hunk 3]
```diff
@@ -567,7 +566,7 @@
 
   .oh-markdown-viewer__content {
     .CodeMirror, .CodeMirror-scroll {
-      min-height: initial;
+      min-height: initial;
```
Comment (jeff-phillips-18) [inline frontend/src/styles/operator-editor-page.scss]: nit: Please remove extra spaces. we should probably add stylelint at some point.

**Comment by jozzi** [comment]
Removed rebase issues from CSS file and checked the rest to ensure any other is present

**Pull request closed without merging**